docs/integration.rst

```diff
@@ -313,6 +313,49 @@ S3ToRedshiftTransfer
 
 .. autoclass:: airflow.operators.s3_to_redshift_operator.S3ToRedshiftTransfer
 
+AWS DynamoDB
+''''''''
```

Comment:
Can you align the `''''`:

```
AWS DynamoDB
''''''''''''
```

Otherwise this will generate warnings at compile time. Thanks.